modules/sugar/src/main/ts/ephox/sugar/api/search/Traverse.ts

```diff
@@ -67,6 +67,27 @@ const parents = (element: SugarElement<Node>, isRoot?: (e: SugarElement<Node>) =
   return ret;
 };
 
+const parentElements = (element: SugarElement<HTMLElement>, isRoot?: (e: SugarElement<Node>) => boolean): SugarElement<HTMLElement>[] => {
```

Comment:
This is more like parentHTMLElements since the return type is not Element[] also you can always make this by just doing `Traverse.parents(..).filter(SugarNode.isHTMLElement)` you could instead of duplicating the code do that it's a small perf diff but you normally don't have like 1000 depth anyway. HTMLElement and just Element can be nested inside each other SVGs can have embedded HTML for example and SVG elements is not HTMLElements.

Also why does it take a element as it's input it would be more generic if you could pass in a node.